modules/veteran_verification/app/controllers/veteran_verification/v0/service_history_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module VeteranVerification
+  module V0
+    class ServiceHistoryController < ApplicationController
+      before_action { authorize :emis, :access? }
```

Comment:
This is going to change from being based on a logged in user to being based on an OAuth token. But I don't object to merging it as-is for the sake of getting the swagger documentation available.